src/envoy/http/authn/filter_context.h

```diff
@@ -53,6 +59,11 @@ class FilterContext : public Logger::Loggable<Logger::Id::filter> {
   HeaderMap* headers() { return headers_; }
   // Accessor to connection
   const Network::Connection* connection() { return connection_; }
+  // Accessor to the filter config
+  const istio::envoy::config::filter::http::authn::v2alpha1::FilterConfig*
+  filterConfig() {
```

Comment:
accessor can be same as variable name, e.g filter_config()

Reply:
Done.